Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphSchemaBuilder.php

```diff
@@ -107,6 +109,30 @@ private function createReferenceRelationTable(): Table
             ->setPrimaryKey(['name', 'position', 'nodeanchorpoint']);
     }
 
+    private function createWorkspaceTable(): Table
+    {
+        $workspaceTable = self::createTable($this->tableNames->workspace(), [
+            DbalSchemaFactory::columnForWorkspaceName('name')->setNotnull(true),
+            DbalSchemaFactory::columnForWorkspaceName('baseWorkspaceName')->setNotnull(false),
+            DbalSchemaFactory::columnForContentStreamId('currentContentStreamId')->setNotNull(true),
```

Comment:
something I noticed while working on another bug is that currently nothing prevents two workspaces having the same contentstream, I think that is dangerous, so we could consider a unique index on this field?

Reply:
Same here, maybe as a followup so we can discuss it extensively?

Reply:
-> https://github.com/neos/neos-development-collection/pull/5296